openssh/USAGE.md

```diff
@@ -0,0 +1,270 @@
+
+Additionally, it is advised to **change the default username and password** when building the image because your plan is to expose it to the world.
+
+```
+Structure of quantum-safe remote access using docker containers
+
++------------------+                +----------------------+
+|      Client      |                |         Host         |
+|  +------------+  |                |  +----------------+  |
+|  |            |  |                |  |                |  |
+|  |   Docker   +--------------------->+     Docker     |  |
+|  |            |  |       Port 2222|  |                |  |
+|  +------------+  |                |  +-------+--------+  |
+|                  |                |          |           |
++------------------+                |  Port 22 v           |
+                                    |  +-------+--------+  |
+                                    |  |                |  |
+                                    |  |  sshd on Host  |  |
+                                    |  |                |  |
+                                    |  +----------------+  |
+                                    |                      |
+                                    +----------------------+
+```
+### Set up the server (docker container on target host)
+
+To start the ssh server, meaning the docker container on the host system, follow those instructions:
+- Run the docker image with
+
+       docker run -dit --network host --name oqs-ssh openquantumsafe/openssh
+
+- Or, if you want the container to automatically start with docker
+
+       docker run -dit --network host --name oqs-ssh --restart unless-stopped openquantumsafe/openssh
+
+The `--network host` option will attach the container directly to your host's network, sharing its IP. The sshd in the container is now accessible from the outside using the host's IP address and the specified port (2222 per default).
+
+Be aware that the port 2222 also needs to be open in any firewalls there may be!
+
+If you want to seriously use this image to connect to a machine, it is **strongly advised** to change the default password of the `oqs` user. This can be done in the **running** container with
+```
+docker exec -it oqs-ssh passwd oqs
+```
+Or you build the image yourself with a different default password from the sources on [Github](https://github.com/openquantumsafe/oqs-demos).
+
+#### Enable classical SSH
+
+Because we wan't to be able to connect to our host that does not run OQS-SSH, we first need to enable classical SSH capabilities for the client **on the host system**.
+
+1. After running the image, run a shell with `docker exec -it oqs-ssh /bin/bash`
+
+2. Run `nano /etc/oqs-ssh/ssh_config` and 
```

Comment:
Now, this unfortunately doesn't work: `/etc/oqs-ssh` doesn't exist. Do you mean `/opt/oqs-ssh`?